Android/app/src/main/java/io/github/project_travel_mate/mytrips/FriendsProfileActivity.java

```diff
@@ -200,9 +219,60 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 // return back to trip activity
                 finish();
                 return true;
+            case R.id.action_sign_out:
```

Comment:
The name of the id will also be more relevant, if you create a new menu file

Reply:
Actually there was use of just one item so I used the existing one. But alright I'll do that.

Reply:
It's just that we might add new menu items to profile page. Then, that will cause issue